peps/pep-0791.rst

```diff
@@ -0,0 +1,184 @@
+PEP: 791
+Title: imath --- module for integer-specific mathematics functions
+Author: Sergey B Kirpichev <skirpichev@gmail.com>
+Sponsor: Victor Stinner <vstinner@python.org>
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Created: 12-May-2025
+Python-Version: 3.15
+Post-History: `12-Jul-2018 <https://mail.python.org/archives/list/python-ideas@python.org/thread/YYJ5YJBJNCVXQWK5K3WSVNMPUSV56LOR/>`__,
+              02-Jun-2019,
+              `09-May-2025 <https://discuss.python.org/t/91337>`__,
+
+
+Abstract
+========
+
+This PEP proposes a new module for number-theoretical, combinatorial and other
+functions defined for integer arguments, like
+:external+py3.14:func:`math.gcd` or :external+py3.14:func:`math.isqrt`.
+
+Suitable functions must be computed exactly, given sufficient time and memory.
```

Comment:
```suggestion
```
This should not be in the abstract.  _The abstract summarizes the PEP_.  Typically everything in the abstract can be found in the PEP, in expanded form.  This sentence doesn't even show up in the PEP.